### PR TITLE
Added keyword highlighting docs

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for https://github.com/mattermost/mattermost/pull/24050

Ticket https://mattermost.atlassian.net/browse/MM-53101

### CODE REVIEW

File: source/preferences/manage-your-notifications.rst [hunk 1]
```diff
@@ -103,7 +103,12 @@ On mobile, select the **Settings** |gear| icon and tap **Notifications**.
 
         **Customize notification keywords**
 
-        You can customize any additional non case-sensitive keywords to trigger notifications. For example, you can receive notifications for all messages and threads related to a specific topic, project name, or customer. Separate multiple keywords using commas or by pressing :kbd:`Tab`, and use :kbd:`Backspace` to manage keywords.
+        All Mattermost users can customize any additional non case-sensitive keywords to trigger notifications. For example, you can receive notifications for all messages and threads related to a specific topic, project name, or customer. Separate multiple keywords using commas or by pressing :kbd:`Tab`, and use :kbd:`Backspace` to manage keywords.
+
+        .. include:: ../_static/badges/ent-pro-only.rst
```
Comment (M-ZubairAhmed) [inline source/preferences/manage-your-notifications.rst:108]: @cwarnermm thank you for the PR, will this PR be attaching the screen shot mentioned here

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

File: source/preferences/manage-your-notifications.rst (hunk 1 above)
Comment (cwarnermm) [reply source/preferences/manage-your-notifications.rst:108]: That image is already included in the codebase as one of our plan/deployment badges. Here's what that code renders as in the generated preview:
image

**Comment by M-ZubairAhmed** [comment]
@cwarnermm can i give you some images of the setting for web?

**Comment by cwarnermm** [comment]
Happy to add additional images to the docs. Were you thinking that we'd want to show the product UI in the docs for this feature, @M-ZubairAhmed?

File: source/preferences/manage-your-notifications.rst [hunk 2]
```diff
@@ -103,7 +103,16 @@ On mobile, select the **Settings** |gear| icon and tap **Notifications**.
 
         **Customize notification keywords**
 
-        You can customize any additional non case-sensitive keywords to trigger notifications. For example, you can receive notifications for all messages and threads related to a specific topic, project name, or customer. Separate multiple keywords using commas or by pressing :kbd:`Tab`, and use :kbd:`Backspace` to manage keywords.
+        All Mattermost users can customize any additional non case-sensitive keywords to trigger notifications. For example, you can receive notifications for all messages and threads related to a specific topic, project name, or customer. Separate multiple keywords using commas or by pressing :kbd:`Tab`, and use :kbd:`Backspace` to manage keywords.
+
+        **Highlight keywords without notifications**
```
Comment (johndavidlugtu) [inline source/preferences/manage-your-notifications.rst:108]: Suggest: Passively track keywords (no notification)

**Review by johndavidlugtu — CHANGES_REQUESTED** [review]

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Review by johndavidlugtu — APPROVED** [review]
Thanks

**Comment by M-ZubairAhmed** [comment]
@cwarnermm would it be possible to add these demos for configuring?

Dec-05-2023 12-53-33
Dec-05-2023 12-55-54

**Review by M-ZubairAhmed — APPROVED** [review]
Thank you so much for being patient with my suggestion and working on these :)

**Comment by cwarnermm** [comment]
We make a great team, @M-ZubairAhmed! Thank YOU!

**Pull request merged**